<!DOCTYPE html>
<html lang="en">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Publications</title>
  <meta name="description" content="">

  <link rel="stylesheet" href="/assets/minima.css">
  <link rel="canonical" href="http://localhost:4000/publications/">
  <link rel="alternate" type="application/rss+xml" title="Greg&#39;s Website" href="/feed.xml">
  
  
</head>

  <body>

    <header class="site-header" role="banner">

  <div class="wrapper">
    
    
    <a class="site-title" href="/">Greg&#39;s Website</a>
  
    
      <nav class="site-nav">
        
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
              <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
              <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger">
          
            
            
            <a class="page-link" href="/cv/">My CV</a>
            
          
            
            
            <a class="page-link" href="/publications/">Publications</a>
            
          
            
            
          
            
            
          
            
            
          
        </div>
      </nav>
    
  </div>
</header>

    <main class="page-content" aria-label="Content">
      <div class="wrapper">
        <h3 id="2020">2020</h3>

<ul class="bibliography"><li><span id="Tsiola2020">Tsiola A, Michoud G, Fodelianakis S, Karakassis I, Kotoulas G, Pavlidou A, Pavloudi C, Pitta P, Simboura N, Daffonchio D, and Tsapakis M. 2020. Viral Metagenomic Content Reflects Seawater Ecological Quality in the Coastal Zone. <i>Viruses</i> <b>12</b>: 806.</span>

<div id="Tsiola2020-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Tsiola2020-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.3390/v12080806">10.3390/v12080806</a></li>
    
  </ul>

  
  <p id="Tsiola2020-abstract" class="collapse">Viruses interfere with their host’s metabolism through the expression of auxiliary metabolic genes (AMGs) that, until now, are mostly studied under large physicochemical gradients. Here, we focus on coastal marine ecosystems and we sequence the viral metagenome (virome) of samples with discrete levels of human-driven disturbances. We aim to describe the relevance of viromics with respect to ecological quality status, defined by the classic seawater trophic index (TRIX). Neither viral (family level) nor bacterial (family level, based on 16S rRNA sequencing) community structure correlated with TRIX. AMGs involved in the Calvin and tricarboxylic acid cycles were found at stations with poor ecological quality, supporting viral lysis by modifying the host’s energy supply. AMGs involved in “non-traditional” energy-production pathways (3HP, sulfur oxidation) were found irrespective of ecological quality, highlighting the importance of recognizing the prevalent metabolic paths and their intermediate byproducts. Various AMGs explained the variability between stations with poor vs. good ecological quality. Our study confirms the pivotal role of the virome content in ecosystem functioning, acting as a “pool” of available functions that may be transferred to the hosts. Further, it suggests that AMGs could be used as an ultra-sensitive metric of energy-production pathways with relevance in the vulnerable coastal zone and its ecological quality.</p>
  

</div></li>
<li><span id="Duarte2020">Duarte CM, Røstad A, Michoud G, Barozzi A, Merlino G, Delgado-Huertas A, Hession BC, Mallon FL, Afifi AM, and Daffonchio D. 2020. Discovery of Afifi, the shallowest and southernmost brine pool reported in the Red Sea. <i>Scientific Reports</i> <b>10</b>: 910.</span>

<div id="Duarte2020-materials">
  <ul class="nav nav-pills">
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1038/s41598-020-57416-w">10.1038/s41598-020-57416-w</a></li>
    
  </ul>

  

</div></li></ul>

<h3 id="2019">2019</h3>

<ul class="bibliography"><li><span id="Zeaiter2019">Zeaiter Z, Marasco R, Booth JM, Prosdocimi EM, Mapelli F, Callegari M, Fusi M, Michoud G, Molinari F, Daffonchio D, Borin S, and Crotti E. 2019. Phenomics and Genomics Reveal Adaptation of Virgibacillus dokdonensis Strain 21D to Its Origin of Isolation, the Seawater-Brine Interface of the Mediterranean Sea Deep Hypersaline Anoxic Basin Discovery. <i>Frontiers in Microbiology</i> <b>10</b>: 1034.</span>

<div id="Zeaiter2019-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Zeaiter2019-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.3389/fmicb.2019.01304">10.3389/fmicb.2019.01304</a></li>
    
  </ul>

  
  <p id="Zeaiter2019-abstract" class="collapse">The adaptation of sporeformers to extreme environmental conditions is frequently questioned due to their capacity to produce highly resistant endospores that are considered as resting contaminants, not representing populations adapted to the system. In this work, in order to gain a better understanding of bacterial adaptation to extreme habitats, we investigated the phenotypic and genomic characteristics of the halophile Virgibacillus sp. 21D isolated from the seawater-brine interface (SBI) of the MgCl2-saturated deep hypersaline anoxic basin Discovery located in the Eastern Mediterranean Sea. Vegetative cells of strain 21D showed the ability to grow in the presence of high concentrations of MgCl2, such as 14.28% corresponding to 1.5 M. Biolog phenotype MicroArray (PM) was adopted to investigate the strain phenotype, with reference to carbon energy utilization and osmotic tolerance. The strain was able to metabolize only 8.4% of 190 carbon sources provided in the PM1 and PM2 plates, mainly carbohydrates, in accordance with the low availability of nutrients in its habitat of origin. By using in silico DNA-DNA hybridization the analysis of strain 21D genome, assembled in one circular contig, revealed that the strain belongs to the species Virgibacillus dokdonensis. The genome presented compatible solute-based osmoadaptation traits, including genes encoding for osmotically activated glycine-betaine/carnitine/choline ABC transporters, as well as ectoine synthase enzymes. Osmoadaptation of the strain was then confirmed with phenotypic assays by using the osmolyte PM9 Biolog plate and growth experiments. Furthermore, the neutral isoelectric point of the reconstructed proteome suggested that the strain osmoadaptation was mainly mediated by compatible solutes. The presence of genes involved in iron acquisition and metabolism indicated that osmoadaptation was tailored to the iron-depleted saline waters of the Discovery SBI. Overall, both phenomics and genomics highlighted the potential capability of V. dokdonensis 21D vegetative cells to adapt to the environmental conditions in Discovery SBI.</p>
  

</div></li>
<li><span id="Mantilla-Calderon20196520">Mantilla-Calderon D, Plewa MJ, Michoud G, Fodelianakis S, Daffonchio D, and Hong PY. 2019. Water Disinfection Byproducts Increase Natural Transformation Rates of Environmental DNA in Acinetobacter baylyi ADP1. <i>Environmental Science and Technology</i> <b>53</b>: 6520–8.</span>

<div id="Mantilla-Calderon20196520-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Mantilla-Calderon20196520-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1021/acs.est.9b00692">10.1021/acs.est.9b00692</a></li>
    
  </ul>

  
  <p id="Mantilla-Calderon20196520-abstract" class="collapse">The process of natural transformation allows for the stable uptake, integration, and functional expression of extracellular DNA. This mechanism of horizontal gene transfer has been widely linked to the acquisition of antibiotic resistance and virulence factors. Here, we demonstrate that bromoacetic acid (BAA) - a regulated drinking water disinfection byproduct (DBP) - can stimulate natural transformation rates in the model organism Acinetobacter baylyi ADP1. We demonstrate that transformation stimulation in response to BAA is concentration-dependent and is linked to the ability of this compound to generate DNA damage via oxidative stress. In presence of BAA, transcription of recA was upregulated 20-40% compared to the nontreated controls, indicating that this component of the DNA damage response could be associated with the increase in transformation. Other genes associated with DNA translocation across the cytoplasmic membrane (i.e., pilX, comA) did not exhibit increased transcription in the presence of BAA, indicating that the enhancement of transformation is not associated with increased translocation rates of environmental DNA. Overall, these results lead us to speculate that elevated recA transcription levels could lead to increased integration rates of foreign DNA within the recipient cell during DNA repair. Lastly, we show that an artificial DBP cocktail simulating the environmental concentrations of five water DBP classes stimulates natural transformation by almost 2-fold. The results of this study suggest that mutagens like DBPs may play an important role in enhancing the fixation rates of extracellular DNA in the environmental metagenome.</p>
  

</div></li>
<li><span id="Mosqueira2019">Mosqueira MJ, Marasco R, Fusi M, Michoud G, Merlino G, Cherif A, and Daffonchio D. 2019. Consistent bacterial selection by date palm root system across heterogeneous desert oasis agroecosystems. <i>Scientific Reports</i> <b>9</b>: 1–12.</span>

<div id="Mosqueira2019-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Mosqueira2019-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1038/s41598-019-40551-4">10.1038/s41598-019-40551-4</a></li>
    
  </ul>

  
  <p id="Mosqueira2019-abstract" class="collapse">Highly productive conventional agroecosystems are spatially embedded in resource-homogeneous systems and count on generally nutrient-rich soils. On the contrary, desert oases are isolated, the soil is relatively poor, but yet productivity is similar to conventional agroecosystems. Soil dominates over plant as the main factor shaping root-associated microbiomes in conventional agroecosystems. We hypothesize that in desert oasis, the environmental discontinuity, the resource paucity and limited microbial diversity of the soil make the plant a prevailing factor. We have examined the bacterial communities in the root system of date palm (Phoenix dactylifera), the iconic keystone species of the oases, grown in heterogeneous soils across a broad geographic range (22,200 km 2 surface area) of the Sahara Desert in Tunisia. We showed that, regardless of the edaphic conditions and geographic location, the plant invariably selects similar Gammaproteobacteria- and Alphaproteobacteria-dominated bacterial communities. The phylogeny, networking properties and predicted functionalities of the bacterial communities indicate that these two phyla are performing the ecological services of biopromotion and biofertilization. We conclude that in a desert agroecosystem, regardless of the soil microbial diversity baseline, the plant, rather than soil type, is responsible of the bacterial community assembly in its root systems, reversing the pattern observed in conventional agroecosystem.</p>
  

</div></li>
<li><span id="MarieBooth2019">Booth JM, Fusi M, Marasco R, Michoud G, Fodelianakis S, Merlino G, and Daffonchio D. 2019. The role of fungi in heterogeneous sediment microbial networks. <i>Scientific Reports</i> <b>9</b>: 7537.</span>

<div id="MarieBooth2019-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#MarieBooth2019-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1038/s41598-019-43980-3">10.1038/s41598-019-43980-3</a></li>
    
  </ul>

  
  <p id="MarieBooth2019-abstract" class="collapse">While prokaryote community diversity and function have been extensively studied in soils and sediments, the functional role of fungi, despite their huge diversity, is widely unexplored. Several studies have, nonetheless, revealed the importance of fungi in provisioning services to prokaryote communities. Here, we hypothesise that the fungal community plays a key role in coordinating entire microbial communities by controlling the structure of functional networks in sediment. We selected a sediment environment with high niche diversity due to prevalent macrofaunal bioturbation, namely intertidal mangrove sediment, and explored the assembly of bacteria, archaea and fungi in different sediment niches, which we characterised by biogeochemical analysis, around the burrow of a herbivorous crab. We detected a high level of heterogeneity in sediment biogeochemical conditions, and diverse niches harboured distinct communities of bacteria, fungi and archaea. Saprotrophic fungi were a pivotal component of microbial networks throughout and we invariably found fungi to act as keystone species in all the examined niches and possibly acting synergistically with other environmental variables to determine the overall microbial community structure. In consideration of the importance of microbial-based nutrient cycling on overall sediment ecosystem functioning, we underline that the fungal microbiome and its role in the functional interactome cannot be overlooked.</p>
  

</div></li>
<li><span id="Alqahtani2019">Alqahtani MF, Bajracharya S, Katuri KP, Ali M, Ragab A, Michoud G, Daffonchio D, and Saikaly PE. 2019. Enrichment of Marinobacter sp. and Halophilic Homoacetogens at the Biocathode of Microbial Electrosynthesis System Inoculated With Red Sea Brine Pool. <i>Frontiers in Microbiology</i> <b>10</b>: 1–18.</span>

<div id="Alqahtani2019-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Alqahtani2019-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.3389/fmicb.2019.02563">10.3389/fmicb.2019.02563</a></li>
    
  </ul>

  
  <p id="Alqahtani2019-abstract" class="collapse">Homoacetogens are efficient CO2 fixing bacteria using H2 as electron donor to produce acetate. These organisms can be enriched at the biocathode of microbial electrosynthesis (MES) for electricity-driven CO2 reduction to acetate. Studies exploring homoacetogens in MES are mainly conducted using pure or mix-culture anaerobic inocula from samples with standard environmental conditions. Extreme marine environments host unique microbial communities including homoacetogens that may have unique capabilities due to their adaptation to harsh environmental conditions. Anaerobic deep-sea brine pools are hypersaline and metalliferous environments and homoacetogens can be expected to live in these environments due to their remarkable metabolic flexibility and energy-efficient biosynthesis. However, brine pools have never been explored as inocula for the enrichment of homacetogens in MES. Here we used the saline water from a Red Sea brine pool as inoculum for the enrichment of halophilic homoacetogens at the biocathode (−1 V vs. Ag/AgCl) of MES. Volatile fatty acids, especially acetate, along with hydrogen gas were produced in MES systems operated at 25 and 10% salinity. Acetate concentration increased when MES was operated at a lower salinity ∼3.5%, representing typical seawater salinity. Amplicon sequencing and genome-centric metagenomics of matured cathodic biofilm showed dominance of the genus Marinobacter and phylum Firmicutes at all tested salinities. Seventeen high-quality draft metagenome-assembled genomes (MAGs) were extracted from the biocathode samples. The recovered MAGs accounted for 87 ± 4% of the quality filtered sequence reads. Genome analysis of the MAGs suggested CO2 fixation via Wood–Ljundahl pathway by members of the phylum Firmicutes and the fixed CO2 was possibly utilized by Marinobacter sp. for growth by consuming O2 escaping from the anode to the cathode for respiration. The enrichment of Marinobacter sp. with homoacetogens was only possible because of the specific cathodic environment in MES. These findings suggest that in organic carbon-limited saline environments, Marinobacter spp. can live in consortia with CO2 fixing bacteria such as homoacetogens, which can provide them with fixed carbon as a source of carbon and energy.</p>
  

</div></li></ul>

<h3 id="2018">2018</h3>

<ul class="bibliography"><li><span id="Barozzi2018">Barozzi A, Mapelli F, Michoud G, Crotti E, Merlino G, Molinari F, Borin S, and Daffonchio D. 2018. Microbial Diversity and Biotechnological Potential of Microorganisms Thriving in the Deep-Sea Brine Pools. In: Extremophiles. CRC Press.</span>

<div id="Barozzi2018-materials">
  <ul class="nav nav-pills">
    

    
    
  </ul>

  

</div></li>
<li><span id="Marasco2018">Marasco R, Rolli E, Fusi M, Michoud G, and Daffonchio D. 2018. Grapevine rootstocks shape underground bacterial microbiome and networking but not potential functionality. <i>Microbiome</i> <b>6</b>: 3.</span>

<div id="Marasco2018-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Marasco2018-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1186/s40168-017-0391-2">10.1186/s40168-017-0391-2</a></li>
    
  </ul>

  
  <p id="Marasco2018-abstract" class="collapse">\textcopyright The Author(s). 2018. Background: The plant compartments of Vitis vinifera, including the rhizosphere, rhizoplane, root endosphere, phyllosphere and carposphere, provide unique niches that drive specific bacterial microbiome associations. The majority of phyllosphere endophytes originate from the soil and migrate up to the aerial compartments through the root endosphere. Thus, the soil and root endosphere partially define the aerial endosphere in the leaves and berries, contributing to the terroir of the fruit. However, V. vinifera cultivars are invariably grafted onto the rootstocks of other Vitis species and hybrids. It has been hypothesized that the plant species determines the microbiome of the root endosphere and, as a consequence, the aerial endosphere. In this work, we test the first part of this hypothesis. We investigate whether different rootstocks influence the bacteria selected from the surrounding soil, affecting the bacterial diversity and potential functionality of the rhizosphere and root endosphere. Methods: Bacterial microbiomes from both the root tissues and the rhizosphere of Barbera cultivars, both ungrafted and grafted on four different rootstocks, cultivated in the same soil from the same vineyard, were characterized by 16S rRNA high-throughput sequencing. To assess the influence of the root genotype on the bacterial communities’ recruitment in the root system, (i) the phylogenetic diversity coupled with the predicted functional profiles and (ii) the co-occurrence bacterial networks were determined. Cultivation-dependent approaches were used to reveal the plant-growth promoting (PGP) potential associated with the grafted and ungrafted root systems. Results: Richness, diversity and bacterial community networking in the root compartments were significantly influenced by the rootstocks. Complementary to a shared bacterial microbiome, different subsets of soil bacteria, including those endowed with PGP traits, were selected by the root system compartments of different rootstocks. The interaction between the root compartments and the rootstock exerted a unique selective pressure that enhanced niche differentiation, but rootstock-specific bacterial communities were still recruited with conserved PGP traits. Conclusion: While the rootstock significantly influences the taxonomy, structure and network properties of the bacterial community in grapevine roots, a homeostatic effect on the distribution of the predicted and potential functional PGP traits was found.</p>
  

</div></li>
<li><span id="Merlino2018">Merlino G, Barozzi A, Michoud G, Ngugi DK, and Daffonchio D. 2018. Microbial ecology of deep-sea hypersaline anoxic basins. <i>FEMS Microbiology Ecology</i> <b>94</b>: 1–15.</span>

<div id="Merlino2018-materials">
  <ul class="nav nav-pills">
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1093/femsec/fiy085">10.1093/femsec/fiy085</a></li>
    
  </ul>

  

</div></li>
<li><span id="Tsiola2018">Tsiola A, Toncelli C, Fodelianakis S, Michoud G, Bucheli TD, Gavriilidou A, Kagiorgi M, Kalantzi I, Knauer K, Kotoulas G, <i>et al.</i> 2018. Low-dose addition of silver nanoparticles stresses marine plankton communities. <i>Environmental Science: Nano</i> <b>4</b>: 2055–65.</span>

<div id="Tsiola2018-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Tsiola2018-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1039/C8EN00195B">10.1039/C8EN00195B</a></li>
    
  </ul>

  
  <p id="Tsiola2018-abstract" class="collapse">The release of silver nanoparticles (AgNPs) is expected to rise in the near future, with possible negative effects on aquatic life and enhancement of microbial resistance against AgNPs. However, a...</p>
  

</div></li>
<li><span id="Vigani2018">Vigani G, Rolli E, Marasco R, Dell’Orto M, Michoud G, Soussi A, Raddadi N, Borin S, Sorlini C, Zocchi G, and Daffonchio D. 2018. Root bacterial endophytes confer drought resistance and enhance expression and activity of a vacuolar H + -pumping pyrophosphatase in pepper plants. <i>Environmental Microbiology</i>: 1–23.</span>

<div id="Vigani2018-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Vigani2018-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1111/1462-2920.14272">10.1111/1462-2920.14272</a></li>
    
  </ul>

  
  <p id="Vigani2018-abstract" class="collapse">The RNA-guided nuclease CRISPR-Cas9 (clustered regularly interspaced short palindromic repeats-CRISPR associated nuclease 9) and its variants such as nickase Cas9, dead Cas9, guide RNA scaffolds and RNA-targeting Cas9 are convenient and versatile platforms for site-specific genome editing and epigenome modulation. They are easy-to-use, simple-to-design, and capable of targeting multiple loci simultaneously. Given that cancer develops from cumulative genetic and epigenetic alterations, CRISPR-Cas9 and its variants (hereafter referred to as CRISPR-Cas9 systems) hold extensive application potentials in cancer modeling and therapy. To date, they have already been applied to model oncogenic mutations in cell lines1 and in adult animals2 , as well as to combat cancer by disabling oncogenic viruses3 or by manipulating cancer genome4 . Given the importance of epigenome and transcriptome in tumorigenesis, manipulation of cancer epigenome and transcriptome for cancer modeling and therapy is a promising area in the future. Whereas (epi)genetic modifications of cancer microenvironment with CRISPR-Cas9 systems for therapeutic purposes represent another promising area in cancer research. Herein, we introduce the functions and mechanisms of CRISPR-Cas9 systems in genome editing and epigenome modulation, retrospect their applications in cancer modelling and therapy, discuss limitations and possible solutions, and propose future directions, in hope of providing concise and enlightening information for readers interested in this area. This article is protected by copyright. All rights reserved.</p>
  

</div></li></ul>

<h3 id="2017">2017</h3>

<ul class="bibliography"><li><span id="Fodelianakis2016">Fodelianakis S, Moustakas A, Papageorgiou N, Manoli O, Tsikopoulou I, Michoud G, Daffonchio D, Karakassis I, and Ladoukakis ED. 2017. Modified niche optima and breadths explain the historical contingency of bacterial community responses to eutrophication in coastal sediments. <i>Molecular Ecology</i> <b>26</b>: 2006–18.</span>

<div id="Fodelianakis2016-materials">
  <ul class="nav nav-pills">
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1111/mec.13842">10.1111/mec.13842</a></li>
    
  </ul>

  

</div></li>
<li><span id="Mapelli2017">Mapelli F, Barozzi A, Michoud G, Merlino G, Crotti E, and Borin S. 2017. Adaption of Microbial Life to Environmental Extremes. In: Stan-Lotter H, Fendrihan S (Eds). Adaption of Microbial Life to Environmental Extremes. Cham: Springer International Publishing.</span>

<div id="Mapelli2017-materials">
  <ul class="nav nav-pills">
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1007/978-3-319-48327-6">10.1007/978-3-319-48327-6</a></li>
    
  </ul>

  

</div></li>
<li><span id="Mapelli2017a">Mapelli F, Scoma A, Michoud G, Aulenta F, Boon N, Borin S, Kalogerakis N, and Daffonchio D. 2017. Biotechnologies for Marine Oil Spill Cleanup: Indissoluble Ties with Microorganisms. <i>Trends in Biotechnology</i> <b>35</b>: 860–70.</span>

<div id="Mapelli2017a-materials">
  <ul class="nav nav-pills">
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1016/j.tibtech.2017.04.003">10.1016/j.tibtech.2017.04.003</a></li>
    
  </ul>

  

</div></li></ul>

<h3 id="2016">2016</h3>

<ul class="bibliography"><li><span id="Callac2016">Callac N, Oger P, Lesongeur F, Rattray J, Vannier P, Michoud G, Beauverger M, Gayet N, Rouxel O, Jebbar M, and Godfroy A. 2016. Pyrococcus kukulkanii sp. nov., a novel hyperthermophilic piezophilic archaeon isolated from a deep-sea hydrothermal vent at the Guaymas Basin. <i>International journal of systematic and evolutionary microbiology</i> <b>66</b>: 3142–9.</span>

<div id="Callac2016-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Callac2016-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1099/ijsem.0.001160">10.1099/ijsem.0.001160</a></li>
    
  </ul>

  
  <p id="Callac2016-abstract" class="collapse">A novel hyperthermophilic, piezophilic anaerobic archaeon, designated as NCB100T, was isolated from a hydrothermal vent flange fragment collected in the Guaymas basin at the hydrothermal vent site named "Rebecca’s Roost" at a depth of 1997 m. Enrichment and isolation were done at 100°C under atmospheric pressure. Cells of strain NCB100T are highly motile irregular cocci with a diameter of ~1 µm. Growth was recorded between 70 and 112°C (optimum 105°C) and hydrostatic pressures of 0.1-80 MPa (optimum 40-50 MPa). Growth was observed at pH 3.5-8.5 (optimum 7), and in 1.5% - 7% NaCl (optimum at 2.5-3%). Strain NCB100T was strictly anaerobic chemoorganoheterotroph and grew on complex proteinaceous substrates such as yeast extract, peptone or tryptone, as well as on glycogen and starch. Elemental sulfur was required for growth and was reduced to hydrogen sulfide. The fermentations products from complex proteinaceous substrates are gluconic acid, pyruvate, CO2 and H2. The G+C content of the genomic DNA was 41.3%. Phylogenetic analysis of the 16S rRNA genes revealed that strain NCB100T belongs to the genus Pyrococcus, showing 99% of similarity with the other described Pyrococcus. Based on the physiological characteristics, G+C content, similarity level between ribosomal proteins and ANI value of 79%, strain NCB100T represents a novel species for which, the name of Pyrococcus kukulkanii sp. nov. is proposed. The type strain is NCB100T (=DSM 101590; Souchothèque de Bretagne BG1337).</p>
  

</div></li>
<li><span id="Jebbar2016">Jebbar M, Vannier P, Michoud G, and Marteinsson VT. 2016. Exploring the Microbiology of the Deep Sea. In: The Marine Microbiome. Cham: Springer International Publishing.</span>

<div id="Jebbar2016-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Jebbar2016-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1007/978-3-319-33000-6_8">10.1007/978-3-319-33000-6_8</a></li>
    
  </ul>

  
  <p id="Jebbar2016-abstract" class="collapse">In this chapter the current knowledge of the diversity of piezophiles isolated so far is reviewed. The isolated piezophiles originated from high-pressure environments such as the cold deep sea, hydrothermal vents, and crustal rocks. Several " stress " conditions can be experienced in these environments, in particular high hydrostatic pressure (HHP). Discoveries of abundant life in diverse high-pressure environments (deep biosphere) support the existence and an adap-tation of life to HHP. At least 50 piezophilic and piezotolerant Bacteria and Archaea have been isolated from different deep-sea environments but these do not by far cover the large metabolic diversity of known microorganisms thriving in deep biospheres. The field of biology of piezophiles has suffered essentially from the requirements for high-pressure retaining sample containments and culturing labo-ratory equipment, which is technically complicated and expensive. Only a few prototypes of HHP bioreactors have been developed by a number of research groups and this could explain the limited number of piezophiles isolated up till now.</p>
  

</div></li>
<li><span id="Martinez2016">Martinez N, Michoud G, Cario A, Ollivier J, Franzetti B, Jebbar M, Oger P, and Peters J. 2016. High protein flexibility and reduced hydration water dynamics are key pressure adaptive strategies in prokaryotes. <i>Scientific Reports</i> <b>6</b>: 32816.</span>

<div id="Martinez2016-materials">
  <ul class="nav nav-pills">
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1038/srep32816">10.1038/srep32816</a></li>
    
  </ul>

  

</div></li>
<li><span id="Michoud2016">Michoud G and Jebbar M. 2016. High hydrostatic pressure adaptive strategies in an obligate piezophile Pyrococcus yayanosii. <i>Scientific Reports</i> <b>6</b>: 27289.</span>

<div id="Michoud2016-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Michoud2016-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1038/srep27289">10.1038/srep27289</a></li>
    
  </ul>

  
  <p id="Michoud2016-abstract" class="collapse">Pyrococcus yayanosii CH1, as the first and only obligate piezophilic hyperthermophilic microorganism discovered to date, extends the physical and chemical limits of life on Earth. It was isolated from the Ashadze hydrothermal vent at 4,100 m depth. Multi-omics analyses were performed to study the mechanisms used by the cell to cope with high hydrostatic pressure variations. In silico analyses showed that the P. yayanosii genome is highly adapted to its harsh environment, with a loss of aromatic amino acid biosynthesis pathways and the high constitutive expression of the energy metabolism compared with other non-obligate piezophilic Pyrococcus species. Differential proteomics and transcriptomics analyses identified key hydrostatic pressure-responsive genes involved in translation, chemotaxis, energy metabolism (hydrogenases and formate metabolism) and Clustered Regularly Interspaced Short Palindromic Repeats sequences associated with Cellular apoptosis susceptibility proteins.</p>
  

</div></li>
<li><span id="Viterbo2016">Viterbo D, Michoud G, Mosbach V, Dujon B, and Richard G-F. 2016. Replication stalling and heteroduplex formation within CAG/CTG trinucleotide repeats by mismatch repair. <i>DNA Repair</i> <b>42</b>: 94–106.</span>

<div id="Viterbo2016-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Viterbo2016-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1016/j.dnarep.2016.03.002">10.1016/j.dnarep.2016.03.002</a></li>
    
  </ul>

  
  <p id="Viterbo2016-abstract" class="collapse">Trinucleotide repeat expansions are responsible for at least two dozen neurological disorders. Mechanisms leading to these large expansions of repeated DNA are still poorly understood. It was proposed that transient stalling of the replication fork by the repeat tract might trigger slippage of the newly-synthesized strand over its template, leading to expansions or contractions of the triplet repeat. However, such mechanism was never formally proven. Here we show that replication fork pausing and CAG/CTG trinucleotide repeat instability are not linked, stable and unstable repeats exhibiting the same propensity to stall replication forks when integrated in a yeast natural chromosome. We found that replication fork stalling was dependent on the integrity of the mismatch-repair system, especially the Msh2p-Msh6p complex, suggesting that direct interaction of MMR proteins with secondary structures formed by trinucleotide repeats in vivo, triggers replication fork pauses. We also show by chromatin immunoprecipitation that Msh2p is enriched at trinucleotide repeat tracts, in both stable and unstable orientations, this enrichment being dependent on MSH3 and MSH6. Finally, we show that overexpressing MSH2 favors the formation of heteroduplex regions, leading to an increase in contractions and expansions of CAG/CTG repeat tracts during replication, these heteroduplexes being dependent on both MSH3 and MSH6. These heteroduplex regions were not detected when a mutant msh2-E768A gene in which the ATPase domain was mutated was overexpressed. Our results unravel two new roles for mismatch-repair proteins: stabilization of heteroduplex regions and transient blocking of replication forks passing through such repeats. Both roles may involve direct interactions between MMR proteins and secondary structures formed by trinucleotide repeat tracts, although indirect interactions may not be formally excluded.</p>
  

</div></li></ul>

<h3 id="2015">2015</h3>

<ul class="bibliography"><li><span id="Lossouarn2015">Lossouarn J, Nesbø CL, Mercier C, Zhaxybayeva O, Johnson MS, Charchuck R, Farasin J, Bienvenu N, Baudoux AC, Michoud G, Jebbar M, and Geslin C. 2015. ’Ménage à trois’: A selfish genetic element uses a virus to propagate within Thermotogales. <i>Environmental Microbiology</i> <b>17</b>: 3278–88.</span>

<div id="Lossouarn2015-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Lossouarn2015-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1111/1462-2920.12783">10.1111/1462-2920.12783</a></li>
    
  </ul>

  
  <p id="Lossouarn2015-abstract" class="collapse">Prokaryotic viruses play a major role in the microbial ecology and evolution. However, the virosphere associated with deep-sea hydrothermal ecosystems remains largely unexplored. Numerous instances of lateral gene transfer have contributed to the complex and incongruent evolutionary history of Thermotogales, an order well represented in deep-sea hydrothermal vents. The presence of clustered regularly interspaced short palindromic repeats (CRISPR) loci has been reported in all Thermotogales genomes, suggesting that these bacteria have been exposed to viral infections that could have mediated gene exchange. In this study, we isolated and characterized the first virus infecting bacteria from the order Thermotogales, Marinitoga piezophila virus 1 (MPV1). The host, Marinitoga piezophila is a thermophilic, anaerobic and piezophilic bacterium isolated from a deep-sea hydrothermal chimney. MPV1 is a temperate Siphoviridae-like virus with a 43.7 kb genome. Surprisingly, we found that MPV1 virions carry not only the viral DNA but preferentially package a plasmid of 13.3 kb (pMP1) also carried by M. piezophila. This ’ménage à trois’ highlights potential relevance of selfish genetic elements in facilitating lateral gene transfer in the deep-sea biosphere.</p>
  

</div></li>
<li><span id="Vannier2015">Vannier P, Michoud G, Oger P, Marteinsson VP, and Jebbar M. 2015. Genome expression of Thermococcus barophilus and Thermococcus kodakarensis in response to different hydrostatic pressure conditions. <i>Research in Microbiology</i> <b>166</b>: 717–25.</span>

<div id="Vannier2015-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Vannier2015-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1016/j.resmic.2015.07.006">10.1016/j.resmic.2015.07.006</a></li>
    
  </ul>

  
  <p id="Vannier2015-abstract" class="collapse">Transcriptomes were analyzed for two related hyperthermophilic archaeal species, the piezophilic Thermococcus barophilus strain MP and piezosensitive Thermococcus kodakarensis strain KOD1 subjected to high hydrostatic pressures. A total of 378 genes were differentially expressed in T. barophilus cells grown at 0.1, 40 and 70 MPa, whereas 141 genes were differentially regulated in T. kodakarensis cells grown at 0.1 and 25 MPa. In T. barophilus cells grown under stress conditions (0.1 and 70 MPa), 178 upregulated genes were distributed among three clusters of orthologous groups (COG): energy production and conversion (C), inorganic ion transport and metabolism (P) and carbohydrate transport and metabolism (G), whereas 156 downregulated genes were distributed among: amino acid transport and metabolism (E), replication, recombination and repair (L) and nucleotide transport and metabolism (F). The expression of 141 genes was regulated in T. kodakarensis cells grown under stress conditions (25 MPa); 71 downregulated genes belong to three COG: energy production and conversion (C), amino acid transport and metabolism (E) and transcription (K), whereas 70 upregulated genes are associated with replication, recombination and repair (L), coenzyme transport (H) and defense mechanisms (V).</p>
  

</div></li></ul>

<h3 id="2014">2014</h3>

<ul class="bibliography"><li><span id="Peters2014">Peters J, Martinez N, Michoud G, Cario A, Franzetti B, Oger P, and Jebbar M. 2014. Deep Sea Microbes Probed by Incoherent Neutron Scattering Under High Hydrostatic Pressure. <i>Zeitschrift für Physikalische Chemie</i> <b>228</b>: 1121–33.</span>

<div id="Peters2014-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Peters2014-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1515/zpch-2014-0547">10.1515/zpch-2014-0547</a></li>
    
  </ul>

  
  <p id="Peters2014-abstract" class="collapse">The majority of the biosphere is a high pressure environment. Around 70% of the marine biosphere lies at depths below 1000m, i.e. at pressures of 100 bars or higher. To survive in these environments, deep-biosphere organisms have adapted to life at high pressure. In vitro studies showed that the activity of certain proteins originating from deep-sea organisms is less affected by high pressure than that of enzymes from surface organisms . However, the genetic and structural bases for this increased pressure resistance are still unknown. Elastic incoherent neutron scattering studies, which provide access to information about molecular dynamics, constitute a very promising approach to decipher the structural adaptation in proteins living under high pressure. This approach has been used in the past to investigate the adaptation of biological systems to temperature and salinity and proved to be essential and complementary to structural studies. Here first investigations of high pressure effects on cell dynamics are presented using Thermococcales as models.</p>
  

</div></li>
<li><span id="Thiel2014">Thiel A, Michoud G, Moalic Y, Flament D, and Jebbar M. 2014. Genetic manipulations of the hyperthermophilic piezophilic archaeon Thermococcus barophilus. <i>Applied and Environmental Microbiology</i> <b>80</b>: 2299–306.</span>

<div id="Thiel2014-materials">
  <ul class="nav nav-pills">
    
    <li><a data-toggle="collapse" href="#Thiel2014-abstract">Abstract</a></li>
    

    
    
    <li>DOI: <a href="http://dx.doi.org/10.1128/AEM.00084-14">10.1128/AEM.00084-14</a></li>
    
  </ul>

  
  <p id="Thiel2014-abstract" class="collapse">In this study, we developed a gene disruption system for Thermococcus barophilus using simvastatin for positive selection and 5-fluoroorotic acid (5-FOA) for negative selection or counterselection to obtain markerless deletion mutants using single- and double-crossover events. Disruption plasmids carrying flanking regions of each targeted gene were constructed and introduced by transformation into wild-type T. barophilus MP cells. Initially, a pyrF deletion mutant was obtained as a starting point for the construction of further markerless mutants. A deletion of the hisB gene was also constructed in the UBOCC-3256 (\DeltapyrF) background, generating a strain (UBOCC-3260) that was auxotrophic for histidine. A functional pyrF or hisB allele from T. barophilus was inserted into the chromosome of UBOCC-3256 (\DeltapyrF) or UBOCC-3260 (\DeltapyrF \DeltahisB), allowing homologous complementation of these mutants. The piezophilic genetic tools developed in this study provide a way to construct strains with multiple genetic backgrounds that will allow further genetic studies for hyperthermophilic piezophilic archaea.</p>
  

</div></li></ul>

      </div>
    </main>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">Greg&#39;s Website</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>
            
              Gregoire Michoud
            
            </li>
            
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/michoug"><span class="icon icon--github"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">michoug</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/GregoireMichoud"><span class="icon icon--twitter"><svg viewBox="0 0 16 16" width="16px" height="16px"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">GregoireMichoud</span></a>

          </li>
          
        </ul>
      </div>

      <div class="footer-col footer-col-3">
        <p></p>
        <p>Updated on the Mon, 14 Sep 2020 18:28:36 +0200<br></p>
      </div>
    </div>

  </div>

</footer>

  </body>

</html>